Modules/Segmentation/LevelSets/include/itkParallelSparseFieldLevelSetImageFilter.h

```diff
@@ -124,16 +124,16 @@ class ITK_TEMPLATE_EXPORT ParallelSparseFieldCityBlockNeighborList
   Print(std::ostream & os, Indent indent) const;
 
 private:
-  char                      m_Pad1[128];
-  unsigned int              m_Size;
-  RadiusType                m_Radius;
-  std::vector<unsigned int> m_ArrayIndex;
-  std::vector<OffsetType>   m_NeighborhoodOffset;
+  char                      m_Pad1[128]{};
+  unsigned int              m_Size{ 2 * Dimension };
+  RadiusType                m_Radius{ 1 };
```

Comment:
The segfault may be related (not tested), better let in ctor as is. It is itk::Size from _Neighborhood_.
`using RadiusType = itk::Size<VDimension>;`

Reply:
Thanks for the investigation @issakomi. Will amend the commit tonight.

Reply:
Done in d741ad0.